oshmem/mca/memheap/base/memheap_base_frame.c

```diff
@@ -59,6 +60,13 @@ static int mca_memheap_base_register(mca_base_register_flag_t flags)
                           MCA_BASE_VAR_SCOPE_LOCAL,
                           &mca_memheap_base_config.device_nic_mem_seg_size);
 
+    mca_base_var_register("oshmem", "memheap", "base", "max_segments",
+                          "Maximum number of segments to register per process",
```

Comment:
"Maximum number of shared data segments per process",

Reply:
> "Maximum number of shared data segments per process",

updated